package.json

```diff
@@ -84,6 +84,7 @@
     "@hmpps-book-secure-move-frameworks/1.5.1": "git+https://github.com/ministryofjustice/hmpps-book-secure-move-frameworks.git#v1.5.1",
     "@hmpps-book-secure-move-frameworks/1.5.2": "git+https://github.com/ministryofjustice/hmpps-book-secure-move-frameworks.git#v1.5.2",
     "@hmpps-book-secure-move-frameworks/1.5.3": "git+https://github.com/ministryofjustice/hmpps-book-secure-move-frameworks.git#v1.5.3",
+    "@hmpps-book-secure-move-frameworks/2.2.0": "git+https://github.com/ministryofjustice/hmpps-book-secure-move-frameworks.git#youth-risk-assessment-updated",
```

Comment:
Obviously a line to replace with a proper version, once the next version of the framework is available. Noting it down here, so you can have the pleasure of dismissing this comment once that's done :)